src/ui/components/Events/Events.tsx

```diff
@@ -97,6 +97,7 @@ function Events() {
                   currentTime={currentTime}
                   executionPoint={executionPoint!}
                   jumpToCodeAnnotation={jumpToCodeEntriesPerEvent[event.point]}
+                  jumpToCodeLoadingStatus={annotationsStatus === "resolved" ? "success" : "loading"}
```

Comment:
Only checking `annotationsStatus === "resolved"` means that we might show "loading" even after the request _fails_.

I think you should swap this (and other places in the file where we have similar checks) to:
```js
annotationsStatus === STATUS_PENDING ? "loading" : "complete"
```